slack-base/src/main/java/com/hubspot/slack/client/models/ReactionIF.java

```diff
@@ -0,0 +1,22 @@
+package com.hubspot.slack.client.models;
+
+import java.util.List;
+
+import org.immutables.value.Value.Immutable;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.hubspot.immutables.style.HubSpotStyle;
+
+@Immutable
+@HubSpotStyle
+@JsonNaming(SnakeCaseStrategy.class)
+public interface ReactionIF {
+  @JsonProperty("name")
+  String getName();
+  @JsonProperty("count")
+  Integer getCount();
+  @JsonProperty("users")
```

Comment:
We tend to use `@JsonProperty` only if the name of the method is different from JSON key. 
In this case, we don't need to name each property explicitly since for example `count` is automatically deserialized and stored in `getCount`. That's the magic of the immutables.